src/utils/DateUtils.ts

```diff
@@ -24,6 +25,13 @@ class DateUtils {
       DateUtils.DATE_FORMAT_OPTIONS
     )
   }
+
+  static getCredentialStatus(expirationDate: number): CredentialStatusEnum {
+    return !expirationDate || expirationDate >= new Date().valueOf() / 1000
```

Comment:
just an improvement we could make, instead of the 1000 value that was already there. we should use DateUtils.EPOCH_MILLISECONDS

Reply:
fixed.